pkg/controller/populators/clone-populator.go

```diff
@@ -57,6 +57,13 @@ const (
 	cloneFinalizer = "cdi.kubevirt.io/clonePopulator"
 )
 
+var desiredCloneAnnotations []string
+
+func init() {
+	desiredCloneAnnotations = append(desiredCloneAnnotations, desiredAnnotations...)
```

Comment:
I think you can just add the `AnnCloneOf` to the desiredannotations list.. it wont hurt anything, maybe the name is not the best and should change maybe you have better name